rero_ils/modules/items/api/record.py

```diff
@@ -72,6 +77,18 @@ def extended_validation(self, **kwargs):
         if len(note_types) != len(set(note_types)):
             return _('Can not have multiple notes of same type.')
 
+        # check temporary item type data
+        tmp_itty = self.get('temporary_item_type')
+        if tmp_itty:
```

Comment:
Does this include also the possibility that a librarian changes the `item_type` to equal the `temporary_item_type`?

Reply:
Yes it's cover. If librarian change default `itty` to use the same value than `tmp_itty` this validation rules will raised an error. In this case, the solution for librarian is to remove the `tmp_itty` value(s)